### PR TITLE
Port CUDAStream to Go

### PR DESCRIPTION
**Pull request by Yancey1989**
related #327

### CODE REVIEW

File: cgotorch/build.sh [hunk 1]
```diff
@@ -12,6 +12,7 @@ GLIBCXX_USE_CXX11_ABI="1"
 LOAD="force_load"
 LIB_SUFFIX="so"
 INSTALL_NAME=""
+CGTORCH_BUILD_ARGS=""
```
Comment (wangkuiyi) [inline cgotorch/build.sh:15]: CGTORCH_BUILD_ARGS => 
CGOTORCH_BUILD_ARGS or
CUDA_FLAGS

**Review by wangkuiyi — COMMENTED** [review]
It seems that we need to complete this PR with more test cases.

File: cgotorch/build.sh (hunk 1 above)
Comment (Yancey1989) [reply cgotorch/build.sh]: done

**Comment by Yancey1989** [comment]
> It seems that we need to complete this PR with more test cases.

Some code should run on CUDA  device, I  have tested them on P100.

**Review by wangkuiyi — APPROVED** [review]

**Pull request merged**